charts/thoras/values.yaml

```diff
@@ -60,6 +60,19 @@ thorasApiServer:
   port: 443
   logLevel: "debug"
 
+thorasApiServerV2:
+  enabled: false
+  containerPort: 8443
+  podAnnotations: {}
+  limits:
+    cpu: 1000m
```

Comment:
[boulder] Can we nix the cpu limit? CPU request is fine but [it's general best practice to avoid limiting CPU](https://home.robusta.dev/blog/stop-using-cpu-limits) unless there's a super specific and unique failure mode you're defending against.

Reply:
Just a heads up, this limit exists for every deployment. Not just the new api service